catroid/src/main/java/org/catrobat/catroid/content/bricks/SetBackgroundByIndexAndWaitBrick.java

```diff
@@ -32,6 +33,11 @@ public SetBackgroundByIndexAndWaitBrick() {
 		wait = EventWrapper.WAIT;
```

Comment:
see my comment in  `SetBackgroundByIndexBrick.java`, but in short -> I think it is preferable to override the `addActionToSequence(..)` method here instead of using the member variable. it just obfuscates the code.